beater/test_approved_es_documents/TestPublishIntegrationEvents.approved.json

```diff
@@ -221,6 +221,15 @@
                     "platform": "Linux"
                 }
             },
+            "http": {
+                "method": "GET",
```

Comment:
this is not an ECS field: https://www.elastic.co/guide/en/ecs/current/ecs-http.html